mqr/baseline_ussd.py

```diff
@@ -0,0 +1,506 @@
+import logging
+
+import vaccine.healthcheck_config as config
+from mqr.utils import rapidpro
+from vaccine.base_application import BaseApplication
+from vaccine.states import Choice, ChoiceState, EndState
+from vaccine.utils import HTTP_EXCEPTIONS, normalise_phonenumber
+
+logger = logging.getLogger(__name__)
+
+
+class Application(BaseApplication):
+    START_SURVEY = "survey_start"
+
+    async def survey_start(self):
+        if config.RAPIDPRO_URL and config.RAPIDPRO_TOKEN:
+            msisdn = normalise_phonenumber(self.inbound.from_addr)
+            urn = f"whatsapp:{msisdn.lstrip(' + ')}"
+
+            if rapidpro:
+                for i in range(3):
+                    try:
+                        contact = rapidpro.get_contacts(urn=urn).first(
+                            retry_on_rate_exceed=True
+                        )
```

Comment:
It's not a good idea to have blocking code running inside the event loop, it's explained well here, even though it's describing for nodejs, it's similar concepts in python: https://nodejs.org/en/docs/guides/dont-block-the-event-loop/#don-t-block-the-event-loop

The easiest way to solve this is to use `run_in_executor`, which runs the sync function in a threadpool: https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.run_in_executor , or by using an async HTTP client.

Reply:
Thanks @rudigiesler , I thought so.

@Buhle79 please ignore this comment for this PR, I'll fix in the next PR. Just going to use the async http client.